src/app/content/studyGuides/components/Filters.spec.tsx

```diff
@@ -161,20 +225,27 @@ describe('Filters', () => {
       yellowCheckbox.props.onChange();
     });
 
-    expect(dispatch).toHaveBeenCalledWith(updateSummaryFilters({
-      colors: { new: [], remove: [HighlightColorEnum.Yellow] },
-    }));
+    expect(dispatch).toHaveBeenCalledWith(
+      replace(mockMatch, {
+        search: `colors=green&colors=blue&colors=purple&locationIds=${chapter.id}&modal=SG`,
+      })
+    );
```

Comment:
@TomWoodward what do I need to add in order to trigger a `locationChange` here rather than `callHistoryMethod`? In the browser a change to the filters dispatches a locationChange which in turn updates the filters in state.

Reply:
@jarosik10 maybe you can advise here?

Reply:
do you really need it to trigger the location change?

i think what is happening is that our convention is to dispatch the `callHistoryMethod` actions, and those are not reduced, the middleware uses them to trigger the actual history method, and then the navigation middleware listens to history actions and dispatches corresponding `changeToLocation` actions

so if you could separately test that the filters dispatch the right history action and then separately that it is reduced properly, that might be easier to do in this setup

the issue is that the test setup doesn't have the navigation middleware, so you really need it to work the same way, you'll have to rig something up to translate the actions from one to the other